Source/JamfProClient.swift

```diff
@@ -80,11 +80,14 @@ struct JamfProClient {
         sendRequest(endpoint: "osxconfigurationprofiles", data: xml.xmlData) { (statusCode, resultData) in
             let success: Bool = (200 <= statusCode && statusCode <= 299)
             if !success {
+                var errMessage = ""
                 if let text = String(data: resultData, encoding: .utf8) {
-                    print("Error (\(statusCode)):\n\(text)")
+                    errMessage = "Error (\(statusCode)):\n\(text)"
                 } else {
-                    print("Unknown error: \(statusCode)")
+                    errMessage = "Unknown error: \(statusCode)"
                 }
+                print(errMessage)
+                Alert().display(header: "Upload failed", message: errMessage)
```

Comment:
```suggestion
                Alert().display(header: "Upload failed:", message: errMessage)
```

Reply:
I don't think we want a colon at the end of the header. The header describes the situation on one line, and then the message gives more detail, and the message already has a colon, so it would look strange to have two colons.

Reply:
I just saw that there is a colon in other messages for example "Attention:".

Reply:
Ah dang, I didn't notice it there.